nixos/modules/security/pam.nix

```diff
@@ -1202,6 +1222,34 @@ in
       };
     };
 
+    security.pam.zfs = {
+      enable = mkOption {
+        default = false;
+        type = types.bool;
+        description = lib.mdDoc ''
+          Enable unlocking and mounting of ZFS home dataset at login.
+        '';
+      };
+
+      homes = mkOption {
+        example = "rpool/home";
+        default = "rpool/home";
+        type = types.str;
+        description = lib.mdDoc ''
+          Prefix of home datasets. This value will be contacenated with
```

Comment:
```suggestion
          Prefix of home datasets. This value will be concatenated with
```

Reply:
Fixed in 5cdfd758d576cceacaee94d290c2e2a6a7d1ee7b.